tools/internal_ci/linux/pull_request/grpc_bazel_on_foundry_opt.sh

```diff
@@ -13,5 +13,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+export UPLOAD_TEST_RESULTS=true
```

Comment:
This is on pull requests too, let's remove the `export UPLOAD_TEST_RESULTS=true`

Reply:
Missed this, thanks